API.md

```diff
@@ -808,6 +810,24 @@ const boolean = Joi.boolean();
 boolean.validate(true, (err, value) => { });
 ```
 
+#### `boolean.truthy(value)`
+
+Allows for additional strings to be considered valid boolean values in addition to default 'truthy' strings: ['true', 'yes', '1', 'on']. Accepts a string or an array of strings.
```

Comment:
Yeah, I figured that was the case @WesTyler. It's definitely just my personal preference here not sure that one method is more correct than the other.

Not sure of the opinion that @Marsup has on this, but I definitely defer to him. Just wanted to state my preference.